### PR TITLE
feat(store): define persistent data structs

### PR DESCRIPTION
**Pull request by paulyoong**
Initial definition of structures that need to be saved in the
persistent store.

Resolves: CAS-819

### CODE REVIEW

File: control-plane/store/src/lib.rs [hunk 1]
```diff
@@ -1,2 +1,3 @@
 pub mod etcd;
 pub mod kv_store;
+pub mod v0;
```
Comment (tiagolobocastro) [inline control-plane/store/src/lib.rs:3]: I wonder if this should be within another submodule, types::v0 or something

**Review by tiagolobocastro — APPROVED** [review]
The Cargo.lock changed so you'll have to change the cargo256

File: control-plane/store/src/lib.rs (hunk 1 above)
Comment (paulyoong) [reply control-plane/store/src/lib.rs]: Good suggestion - done.

**Comment by paulyoong** [comment]
> The Cargo.lock changed so you'll have to change the cargo256

Good spot - done.

**Comment by paulyoong** [comment]
bors merge

**Comment by paulyoong** [comment]
bors merge

**Pull request merged**